pkg/controller/neutronovsagent/neutronovsagent_controller.go

```diff
@@ -380,8 +400,8 @@ func newDaemonset(cr *neutronv1.NeutronOvsAgent, cmName string, configHash strin
                                 MountPropagation: &bidirectional,
                         },
                         {
-                                Name:      "rendered-config-vol",
-                                MountPath: "/mnt",
+                                Name:      "neutron-config-vol",
+                                MountPath: "/etc/neutron",
```

Comment:
So we are going to mount these directly to /etc/neutron? This works but could cause permissions issues in the container. I think that is the reason we typically use kolla_start. Perhaps lets leave this for now, but we should revisit using kolla_start (the default entry point for most RDO containers) so that we have the extra layer of indirection.

Reply:
ack, we'll revisit using kolla_start soon. for now we should have this.